opal/mca/btl/base/btl_base_am_rdma.c

```diff
@@ -409,7 +410,13 @@ static inline int mca_btl_base_am_rdma_advance(mca_btl_base_module_t *btl,
     }
 
     if (send_descriptor) {
-        return btl->btl_send(btl, endpoint, descriptor, mca_btl_base_rdma_tag(hdr->type));
+        /* This descriptor was created with MCA_BTL_DES_SEND_ALWAYS_CALLBACK set, so safe to treat
```

Comment:
You don't need the comment.  The assert says the same thing :).

Reply:
Thank you! removed in new revision